static/js/contextMenu.js

```diff
@@ -0,0 +1,355 @@
+    
+    var $userList = $('<div>', {id: 'userList'});
+    $container.append($userList);
+
+    var $modalControls = $('<div>', {'class': 'display-bottomright'});
+    var $btnCancel = $('<button>', {'class': 'custom-button'});
+    $btnCancel.on('click', function(e) {
+      $(notifyModal).fadeToggle(200);
+    })
+    $btnCancel.text('CANCEL')
+    var $btnNotify = $('<button>', {'class': 'custom-button'});
+    $btnNotify.on('click', function(e) {
+      $.ajax({
+        method: 'POST',
+        url: baseURL +'/mypads/api/notify-users',
+        data: {
+          copiedText: getTextToCopy(),
+          loginsOrEmails: loginOrEmails
+        },
+        success: function(data, textStatus, jqXHR) {
+          if (data.success === true) {
+            console.log('notifying success!');
+          } else {
+            console.log('notifying failed');
+          }
+        }
+      });
+      $(notifyModal).fadeToggle(200);
+    })
+    $btnNotify.text('SEND')
+    $modalControls.append($btnCancel);
+    $modalControls.append('<span style="margin-right:15px;"></span>')
+    $modalControls.append($btnNotify);
+    $container.append($modalControls);
+
+    $modalContent.append($container);
+    $(notifyModal).append($modalContent);
+    $(notifyModal).on('click', function(e) {
+      $(this).fadeToggle(200);
+    });
+    $modalContent.on('click', function(e) {
+      return false;
+    })
+    $btnSelectUser.on('click', function(e) {
+      var loginOrEmail = $emailField.val();
+      if (loginOrEmails.includes(loginOrEmail)) {
+        return;
+      }
+      
+      console.log(baseURL);
+      console.log(loginOrEmail);
+      console.log('sending ajax request');
+      $.ajax({
+        method: 'GET',
+        url: baseURL+'/mypads/api/user-exist/' + loginOrEmail,
+        success: function(data, textStatus, jqXHR) {
+          console.log('response: ');
+          console.log(data);
+          if (data.userExists === true) {
+            $msgUserNotExist.hide();
+            loginOrEmails.push(loginOrEmail);
+            
+            var $selectedUser = $("<span>" + loginOrEmail + "<span style='padding-left:5px; font-size: large' class = 'clickable'>&times;</span></span>");
+            $selectedUser.css('border-radius', "5px");
+            $selectedUser.css('background-color', "darkgray");
+            $selectedUser.css('padding', "5px");
+            $selectedUser.find('.clickable').css('cursor', 'pointer');
+            $selectedUser.find('.clickable').on('click', function(e) {
+              var index = loginOrEmails.indexOf(loginOrEmail);
+              if (index > -1) {
+                loginOrEmails.splice(index, 1);
+              }
+              $selectedUser.remove();
+            });
+            $userList.append($selectedUser);
+          } else {
+            $msgUserNotExist.show();
+          }
+        }
+      });
+    })
+
+    $(notifyModal).css({
```

Comment:
Not sure if feasible or where to implement, but I think styling would be better suited as a static css file.  And then just assign the class to the element instead.